pkg/tests/ossm/initContainer_test.go

```diff
@@ -15,42 +15,79 @@
 package ossm
 
 import (
-	_ "embed"
-	"strings"
 	"testing"
 
-	"github.com/maistra/maistra-test-tool/pkg/util"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/hack"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
 	"github.com/maistra/maistra-test-tool/pkg/util/test"
 )
 
-const (
-	initContainerNS         = "bookinfo"
-	initContainerGoldString = "init worked"
-)
+func TestInitContainerNotRemovedDuringInjection(t *testing.T) {
+	const ns = "bookinfo"
+	const goldString = "[init worked]"
+	const podSelector = "app=sleep-init"
 
-var (
-	//go:embed yaml/deployment-sleep-init.yaml
-	testInitContainerYAML string
-)
+	test.NewTest(t).Id("T33").Groups(test.Full).Run(func(t test.TestHelper) {
 
-func cleanupInitContainer() {
-	util.KubeDeleteContents(initContainerNS, testInitContainerYAML)
-}
+		hack.DisableLogrusForThisTest(t)
+
+		t.Log("Checking init container not removed during sidecar injection.")
+
+		t.Cleanup(func() {
+			oc.DeleteFromString(t, ns, testInitContainerYAML)
+			oc.DeleteNamespace(t, ns)
```

Comment:
No need to delete resources in a namespace if you then delete the entire namespace.  Also, some tests might expect the bookinfo namespace to exist, so instead of deleting it, it's best to recreate it (at least until we fix all the other tests and make them create whatever namespaces they need).

```suggestion
			oc.RecreateNamespace(t, ns)
```

Reply:
@monroe-dylan-m this PR just needs this small change and it can be merged.